cmd/geth/consolecmd.go

```diff
@@ -137,6 +137,9 @@ func remoteConsole(ctx *cli.Context) error {
 			} else if ctx.GlobalBool(utils.MumbaiFlag.Name) {
 				homeDir, _ := os.UserHomeDir()
 				path = filepath.Join(homeDir, "/.bor/data")
+			} else if ctx.GlobalBool(utils.BorMainnetFlag.Name) {
```

Comment:
Since mumbai and mainnet have same path, Lets use OR here and this condition with mumbai flag only

Reply:
Done!